monai/apps/detection/transforms/array.py

```diff
@@ -122,7 +129,148 @@ def __call__(self, boxes: NdarrayOrTensor) -> NdarrayOrTensor:
         Standard mode is "xyxy" or "xyzxyz",
         representing box format of [xmin, ymin, xmax, ymax] or [xmin, ymin, zmin, xmax, ymax, zmax].
 
+        Args:
+            boxes: source bounding boxes, Nx4 or Nx6 torch tensor or ndarray. The box mode is assumed to be ``StandardMode``
+
         Returns:
             bounding boxes with standard mode, with same data type as ``boxes``, does not share memory with ``boxes``
         """
         return convert_box_to_standard_mode(boxes, mode=self.mode)
+
+
+class AffineBox(Transform):
+    """
+    Applys affine matrix to the boxes
+    Args:
+        invert_affine: whether to apply inverse affine matrix
```

Comment:
I don't get the usage of this `invert_affine` arg, maybe no need it in this array transform?

Thanks.

Reply:
It's used in dictionary, control __call__ and __invert__. But it's fine to remove, I could put inverse affine computation in dictionary